app/src/app/components/sidebar/Evaluation.tsx

```diff
@@ -0,0 +1,325 @@
+  }
+
+  if (error) {
+    return (
+      <div>
+        <h1>Summary Statistics</h1>
+        <p>There was an error loading the summary statistics.</p>
+      </div>
+    );
+  }
+  const rows = unassigned && showUnassigned ? [...data.results, unassigned] : data.results;
+  return (
+    <div className="w-full">
+      <Flex align="center" gap="3" mt="1">
+        {modeButtonConfig.map((mode, i) => (
+          <Button
+            key={i}
+            variant={mode.value === evalMode ? 'solid' : 'outline'}
+            onClick={() => setEvalMode(mode.value)}
+          >
+            {mode.label}
+          </Button>
+        ))}
+        {isLoading && <Spinner />}
+      </Flex>
+
+      <Flex align="center" gap="3" mt="1">
+        <CheckboxGroup.Root
+          defaultValue={[]}
+          orientation="horizontal"
+          name="evaluation-options"
+          value={[
+            // showAverages ? 'averages' : '',
+            // showStdDev ? 'stddev' : '',
+            colorBg ? 'colorBg' : '',
+            showUnassigned ? 'unassigned' : '',
+          ]}
+        >
+          <CheckboxGroup.Item value="unassigned" onClick={() => setShowUnassigned(v => !v)}>
+            Show Unassigned Population
+          </CheckboxGroup.Item>
+          {/* <CheckboxGroup.Item value="averages" onClick={() => setShowAverages(v => !v)}>
+            Show Zone Averages
+          </CheckboxGroup.Item>
+          <CheckboxGroup.Item value="stddev" onClick={() => setShowStdDev(v => !v)}>
+            Show Zone Std. Dev.
+          </CheckboxGroup.Item> */}
+          <CheckboxGroup.Item value="colorBg" onClick={() => setColorBg(v => !v)}>
+            <Flex gap="3">
+              <p>Color Cells By Values</p>
+              {/* {colorByStdDev && (
+                <span>
+                  {Object.entries(stdDevColors)
+                    .sort((a, b) => +a[0] - +b[0])
+                    .map(([stdev, backgroundColor], i) => (
+                      <span
+                        className="inline-flex items-center justify-center size-6"
+                        style={{backgroundColor}}
+                      >
+                        {+stdev > 0 ? `+${stdev}`: stdev}
+                      </span>
+                    ))}
+                </span>
+              )} */}
+            </Flex>
+          </CheckboxGroup.Item>
+        </CheckboxGroup.Root>
+      </Flex>
+      <div className="overflow-x-auto text-sm">
+        <table className="min-w-full border-collapse">
+          <thead>
+            <tr className="bg-gray-50 border-b">
+              <th className="py-2 px-4 text-left font-semibold">Zone</th>
+              {columnConfig.map((f, i) => (
+                <th className="py-2 px-4 text-right font-semibold" key={i}>
+                  {f.label}
+                </th>
+              ))}
+            </tr>
+          </thead>
+          <tbody>
+            {/* {!!(averages && showAverages) && (
```

Comment:
Q: these were some options we ended up not using -- should we clean up the file or keep these in case we'd want something like them in the future?